formver.compiler-plugin/viper/src/org/jetbrains/kotlin/formver/viper/ast/Exp.kt

```diff
@@ -565,6 +565,27 @@ sealed interface Exp : IntoSilver<viper.silver.ast.Exp> {
             Unfolding(predicateAccess.toSilver(), body.toSilver(), pos.toSilver(), info.toSilver(), trafos.toSilver())
     }
 
+    data class Acc(
+        val field: FieldAccess,
+        val perm: PermExp,
+        val pos: Position = Position.NoPosition,
+        val info: Info = Info.NoInfo,
+        val trafos: Trafos = Trafos.NoTrafos,
+    ) : Exp {
+        override val type = field.type
```

Comment:
Is this true?  I think it should always be a boolean, since the access permission is a boolean.  (Or did we do something different for forall?)

Reply:
Right that makes sense, for `Forall` it was also a boolean, but I had modeled this after the field access